docs/installing.md

```diff
@@ -31,14 +56,14 @@ If you don't use *screen*, open 4 terminal windows and run in each one of them:
   4. `make alice-desktop-local` or if you want to run Alice as a daemon: `make alice-daemon-local`
```

Comment:
I think we should move step 3 of these instructions (registering the arbitrator) to a new sentence after the numbered list. It applies whether using screen or not, so this would instruct to register as a last step regardless.